Mathlib/Algebra/Order/GroupWithZero/Unbundled.lean

```diff
@@ -767,162 +866,218 @@ theorem lt_mul_of_lt_of_one_le_of_nonneg [PosMulMono α] (h : b < c) (ha : 1 ≤
     b < c * a :=
   h.trans_le <| le_mul_of_one_le_right hc ha
 
+set_option linter.deprecated false
+
 /-- Assumes left covariance. -/
+@[deprecated Left.one_le_mul₀ (since := "2024-10-10")]
 theorem Left.one_le_mul_of_le_of_le [PosMulMono α] (ha : 1 ≤ a) (hb : 1 ≤ b) (a0 : 0 ≤ a) :
     1 ≤ a * b :=
   le_mul_of_le_of_one_le_of_nonneg ha hb a0
 
 /-- Assumes left covariance. -/
+@[deprecated (since := "2024-10-10")]
```

Comment:
Can you add the replacement theorems to the `deprecated` code block here and in other places they're missing?